charts/super-agent-deployment/values.yaml

```diff
@@ -97,18 +97,65 @@ fedramp:
 verboseLog:
 
 config:
-  # -- Set if the configMap is going to be created by this chart or the user will provide its own.
-  create: true
-  # -- The name the configMap is going to have. If create is set to false, the name of an existing configMap that will be used to config the Super Agent.
-  name: ""
-  # -- The key in the configMap that has the configuration for the Super Agent.
-  key: ""
-  # -- Here you can set New Relic' Super Agent configuration.
+  # -- Configuration for the Super Agent.
+  # @default -- See `values.yaml`
+  superAgent:
+    # -- Set if the configMap is going to be created by this chart or the user will provide its own.
+    create: true
```

Comment:
why do we have a create option for these CMs and not for the ones below? Should we move that create one level up and make it control both?

Reply:
The difference is that this creates a config map always, with the license key and such, while the fleet configuration only creates config maps if there is something to configure.

If there is fleet configuration there is a config map and a mount point.

If a user wants to provide their own configuration, there are a `extraMounts` and `extraVolumes` fields on the `vaules.yaml`.